setup.cfg

```diff
@@ -35,6 +35,7 @@ package_dir =
   = src
 packages = find:
 install_requires =
+    ansible-builder >=1,<2
```

Comment:
(educational, optional)  There's a possibility to use `~=` as an alias to what's used here like this. If we don't know if `ansible-builder` is going to be following strict SemVer, it _may be_ better to use it (per your discretion):
```suggestion
    ansible-builder ~= 1
```
Another possibility to improve this is to add spacing to version specifiers — it's a bit easier/nicer to consume when it's sparse:
```suggestion
    ansible-builder >=1, <2
```

Reply:
FIxed